src/arcaflow_plugin_sdk/schema.py

```diff
@@ -2636,21 +2645,51 @@ class OneOfStringSchema(_JSONSchemaGenerator, _OpenAPIGenerator):
     ...                 )
     ...             }
     ...         )
-    ...     }
+    ...     },
+    ...     discriminator_inlined=False,
     ... )
     """  # noqa: E501
 
     types: Dict[str, typing.Annotated[_OBJECT_LIKE, discriminator("type_id")]]
+    discriminator_inlined: typing.Annotated[
+        bool,
+        _name("Discriminator field inlined"),
+        _description(
+            "Whether or not the discriminator is inlined in the underlying"
+            " objects' schema"
+        ),
```

Comment:
"objects' schemas" sounds like each object has multiple schemas. I left schema singular because I expected each object to be represented by only one schema in the union. I guess it is possible for each object to be represented by more than one schema.

Reply:
> "objects' schemas" sounds like each object has multiple schemas.

`objects'` is plural; so, unless all the objects have the same schema (which seems unlikely, since that would suggest that we don't need a OneOf), there are multiple schemas in play here, in which case "schemas" should be plural.

How about
> True if the discriminator is a field in each of the underlying objects' schemas

or

> True if the discriminator is a field in each schema of the underlying objects

Reply:
did it